src/flow_solver/flow_solver_cases/periodic_cube_flow.cpp

```diff
@@ -33,15 +33,19 @@ PeriodicCubeFlow<dim, nstate>::PeriodicCubeFlow(const PHiLiP::Parameters::AllPar
 template <int dim, int nstate>
 std::shared_ptr<Triangulation> PeriodicCubeFlow<dim,nstate>::generate_grid() const
 {
+
+    const int mpi_rank = dealii::Utilities::MPI::this_mpi_process(MPI_COMM_WORLD);
+    dealii::ConditionalOStream pcout(std::cout, mpi_rank==0);
```

Comment:
delete these and use `this->pcout`; already exists in the base class

Reply:
Fixed this in the recent commit.